versioned_docs/version-8.4/components/modeler/web-modeler/play-your-process.md

```diff
@@ -47,6 +47,19 @@ Incidents are raised as they would in any Camunda cluster. Use the variables and
 
 To start a new process instance, click on the **Restart process** button on the start event, or return to the definition view by clicking **View all** on the top banner.
 
+## Rewind a process
```

Comment:
I think I'm just curious how to actually rewind the process without a screenshot -- do I click an icon? How do I select where I rewind this to?

Reply:
Good point. The screenshot should include the rewind icon, and the docs should mention that there's a rewind button

Reply:
screenshot added 👍